www/src/data/sidebars/doc-links.yaml

```diff
@@ -1,128 +1,221 @@
-- title: Quick Start
+- title: Quick start
   items:
-    - title: Getting Started
+    - title: Quick start
       link: /docs/
-    - title: Starters
-      link: /docs/gatsby-starters/
+    - title: Overview of the Docs*
+      link: /docs/overview/
+    - title: Main Tutorial (step-by-step)
+      link: /tutorial/
+- title: Deploying & Hosting
+  items:
+    - title: Preparing site for deployment*
+      link: /docs/preparing-for-deployment/
     - title: Deploying
       link: /docs/deploy-gatsby/
-    - title: Community
-      link: /docs/community/
-    - title: Awesome Gatsby
-      link: /docs/awesome-gatsby/
-- title: Core Concepts
-  items:
-    - title: Building with Components
-      link: /docs/building-with-components/
-    - title: Lifecycle APIs
-      link: /docs/gatsby-lifecycle-apis/
-    - title: Plugins
-      link: /docs/plugins/
-    - title: PRPL Pattern
-      link: /docs/prpl-pattern/
-    - title: Querying Data with GraphQL
-      link: /docs/querying-with-graphql/
+    - title: Path Prefix
+      link: /docs/path-prefix/
+    - title: How Gatsby Works with GitHub pages
+      link: /docs/how-gatsby-works-with-github-pages/
 - title: Guides
   items:
-    - title: Adding a 404 Page
-      link: /docs/add-404-page/
-    - title: Adding Images, Fonts, and Files
-      link: /docs/adding-images-fonts-files/
-    - title: Adding Images to a WordPress Site
-      link: /docs/image-tutorial/
-    - title: Babel.js
-      link: /docs/babel/
-    - title: Browser Support
-      link: /docs/browser-support/
-    - title: Building Apps with Gatsby
+    - title: Environment*
+      link: /docs/environment/
+      items:
+      - title: Basic hardware and software requirements*
+        link: /docs/basic-hardware-software-requirements/
+      - title: Browser Support
+        link: /docs/browser-support/
+      - title: Gatsby on Macs*
+        link: /docs/gatsby-on-macs/
+      - title: Gatsby on Windows
+        link: /docs/gatsby-on-windows/
+    - title: Customization*
+      link: /docs/customization/
+      items:
+        - title: Babel.js
```

Comment:
Do we have a generic verb tense? Perhaps `Using Babel.js`?

Reply:
(or remove verbs from everything)

Reply:
I'd vote for no verbs. If I change the .md title and the link pathway to match the new name, is there anything else that needs to be done? I would have already done that, just thought maybe it'd cause someone else to have to handle redirects and didn't know if anyone has time for that or who to ask to do it!